runtime/src/main/java/org/corfudb/runtime/Restore.java

```diff
@@ -171,4 +199,13 @@ private void verify() {
     private void cleanup() throws IOException {
         FileUtils.deleteDirectory(new File(restoreTempDirPath));
     }
+
+    private void clearAllTables() {
+        TxnContext txn = corfuStore.txn(CORFU_SYSTEM_NAMESPACE);
+        Collection<TableName> tables = corfuStore.listTables(null);
```

Comment:
let's pass an empty list, instead of null

Reply:
We can't do that right? `lisTables(String namespace)` when passing null it will return all tables. From the current code if I pass empty string it will only search for empty namespace.
`.filter(tableName -> namespace == null || tableName.getNamespace().equals(namespace))`

Reply:
that looks weird, I'd have two different methods then, first gets a list of tables, a second one has no parameters and returns all the tables, the current implementation looks a bit convoluted